introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<h3>and available -&gt; and is available</h3><h3>please remove "for inference".</h3><h3>Let's add a title to the section, even without numbering, and then put the current text without bold.</h3><p>Please add: Inference on the models listed in the dropdown menu can be run in network isolation [https://docs.aws.amazon.com/sagemaker/latest/dg/mkt-algo-model-internet-free.html] under VPC settings. However, when running inference on a model specified through HF_MODEL_ID, VPC settings with network isolation will not work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>